src/app.js

```diff
@@ -15,7 +15,11 @@ app.extend({
       if (!Array.isArray(val)) {
         val = [val]
       }
-      this.allRepos = _.map(val, function(n){return _.values(n)})[0]
+      let repos = _.map(val[0], function(value, key) {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'_' is not defined.